setup.py

```diff
@@ -41,6 +49,7 @@ def run_tests(self):
 # Dependencies for the package itself
 DEPENDENCIES = [
     "matplotlib>=3.1.0",
+    "pillow>=6.0.0",  # not our dependency but older versions crash with mpl
```

Comment:
The actually package is `Pillow`, though I don't know if the lowercase P could be an issue. Can't hurt to make the change.

Reply:
@JoschD I just saw that matplotlib has a `Pillow>=6.2.0"` requirement, is this one needed then?

Reply:
Weird. I have an old mpl and old Pillow (5.1.0) installed. 
Only the mpl requirement in the Dependencies.
Then I run `python setup.py install test`.
-> Installs mpl 3.2.1
crashes `ImportError: cannot import name '_imaging' from 'PIL' (/usr/lib/python3/dist-packages/PIL/__init__.py)`

With the explicit pillow dependency it updates pillow and runs fine.
Can you verify?

Reply:
Indeed I can confirm, quite strange. Let's leave the Pillow requirement here.